operator/v1/0000_50_ingress-operator_00-ingresscontroller.crd.yaml

```diff
@@ -981,15 +981,17 @@ spec:
                     description: "threadCount defines the number of threads created
                       per HAProxy process. Creating more threads allows each ingress
                       controller pod to handle more connections, at the cost of more
-                      system resources being used. If this field is empty, the IngressController
-                      will use the default value.  The current default is 4 threads,
-                      but this may change in future releases. \n Setting this field
-                      is generally not recommended. Increasing the number of HAProxy
+                      system resources being used. HAProxy currently supports up to
+                      64 threads. If this field is empty, the IngressController will
+                      use the default value.  The current default is 4 threads, but
+                      this may change in future releases. \n Setting this field is
+                      generally not recommended. Increasing the number of HAProxy
                       threads allows ingress controller pods to utilize more CPU time
                       under load, potentially starving other pods if set too high.
                       Reducing the number of threads may cause the ingress controller
                       to perform poorly."
                     format: int32
+                    maximum: 64
```

Comment:
You're changing a value from previously unbounded? What is the chance that this value is currently set higher than 64 in existing clusters?

Reply:
The field is new in 4.8, but even if someone is running a 4.8 pre-release build and `threadCound` is set above 64, HAProxy exits with a fatal error and the router pods quickly end up in CrashLoopBackOff state, so it shouldn't be possible that a working cluster has `threadCount` set above 64.